client/web/compose/src/components/PageBlocks/ChartBase.vue

```diff
@@ -183,6 +197,10 @@ export default {
         filter = this.liveFilterValue
       }
 
+      if (!this.liveFilterValue && this.predefinedFilters.find(({ value }) => value === this.filter.filter)) {
```

Comment:
This should only happen when you open the live filter and there is no filter yet.
Otherwise it sets it even if you want it to not be filtered.